CMakeLists.txt

```diff
@@ -488,14 +488,15 @@ endif()
 if(WIN32 AND NOT HWLOC_FOUND)
   find_library(HWLOC_DLL "hwloc")
   find_library(HWLOC_LIB "libhwloc.lib")
-  find_path(HWLOC_INC_DIR "hwloc.h")
-  if (HWLOC_LIB AND HWLOC_DLL AND HWLOC_INC_DIR)
+  find_path(HWLOC_INCLUDE_DIRS "hwloc.h")
+  if (HWLOC_LIB AND HWLOC_DLL AND HWLOC_INCLUDE_DIRS)
     set(HWLOC_FOUND 1)
   endif()
 endif()
 
 if(HWLOC_FOUND)
   set(DEFAULT_ENABLE_HWLOC ON CACHE INTERNAL "default hwloc")
+  include_directories(${HWLOC_INCLUDE_DIRS})
```

Comment:
Forgot to rename HWLOC_INC_DIR [here](https://github.com/pocl/pocl/blob/0d934c2756446c1a8effb267502bb53c02517488/lib/CL/devices/topology/CMakeLists.txt#L41)? Once fixed this line should be redundant.

Reply:
Ah yes, updated.